### ISSUE TITLE
Website Accessibility-WCAG 1.3.1-Ensures <li> elements are used semantically-ul[role="navigation"] > li:nth-child(4)

### ISSUE THREAD

**Issue by A11Ybot-OP** [angular/angular#44565]
### Description

Issue 
Ensures ````<li> elements are used semantically````

Target application
https://angular.io/

Path 
````ul[role="navigation"] > li:nth-child(4)````

Snippet
````<li class="ng-star-inserted"><a class="nav-link" href="events" title="Events."><span class="nav-link-inner">Events</span></a></li>````

Fix the following:
List item does not have a ````<ul>, <ol>```` parent element without a role, or a role="list"

Environment
Microsoft Edge version 96.0.1054.62
